idea/resources/META-INF/plugin-common.xml

```diff
@@ -3404,7 +3404,16 @@
                      level="WEAK WARNING"
                      language="kotlin"
     />
-    
+
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.ReplaceNotNullAssertionWithMapNotNullInspection"
+                     displayName="Replace '!!' with 'mapNotNull'"
```

Comment:
`'map' with not-null assertion should be simplified to 'mapNotNull'` looks better.